drivers/bluetooth/hci/ipm_stm32wb.c

```diff
@@ -421,11 +421,15 @@ static void start_ble_rf(void)
 	LL_RCC_SetRTCClockSource(LL_RCC_RTC_CLKSOURCE_LSI);
 #endif
 
-	/* Set RNG on HSI48 */
+#if !STM32_HSI48_ENABLED
+#warning Bluetooth IPM requires HSI48 clock to be enabled
+	/* Deprecated: Set BLE on HSI48 */
```

Comment:
We should clarify that HSI48 should be enabled **using device tree**

Reply:
`#warning Bluetooth IPM requires HSI48 clock to be enabled using device `